includes/admin.form.inc

```diff
@@ -60,12 +84,21 @@ function islandora_memberofsite_admin_form_submit(array $form, array &$form_stat
   switch ($id) {
     case 'edit-submit':
       variable_set('islandora_memberofsite_namespace', $form_state['values']['namespace']);
-      variable_set('islandora_memberofsite_sites', $form_state['values']['sites']);
+      variable_set('islandora_memberofsite_site_model', $form_state['values']['site_model']);
```

Comment:
We've changed the variable stored from a string to an array, so we should call a Drupal update hook to modify (clear?) the existing settings.